vault-kubernetes-sidecar/courseBase.sh

```diff
@@ -0,0 +1,4 @@
+# Install Helm 3 and overwrite Helm 2
```

Comment:
Keep this for now but I'm going to move this into the base `hashicorp-hashistack` image since it's used on several scenarios.

Reply:
That's great.